cardigan/stories/components/Buttons/Buttons.stories.tsx

```diff
@@ -29,7 +29,7 @@ export const control = ControlTemplate.bind({});
 control.args = {
   text: 'something for screenreaders',
   icon: chevron,
-  extraClasses: 'control--light',
+  colorScheme: 'light',
```

Comment:
This hadn't been styled properly for a bit, the former class doesn't exist and we use the `colorScheme` property now. So the story wasn't up to date.